openapi/kas-fleet-manager.yaml

```diff
@@ -951,6 +985,77 @@ components:
           description: Whether connection reauthentication is enabled or not. If set to true, connection reauthentication on the Kafka instance will be required every 5 minutes. The default value is true
           type: boolean
           nullable: true
+    SupportedKafkaInstanceTypesList:
+      allOf:
+        - $ref: "#/components/schemas/List"
+        - type: object
+          example:
+            kind: "SupportedKafkaInstanceTypesList"
+            page: "1"
+            size: "1"
+            total: "1"
+            item:
+              $ref: "#/components/examples/SupportedKafkaInstanceTypeExample"
+          properties:
+            items:
+              type: array
+              items:
+                allOf:
+                  - $ref: "#/components/schemas/SupportedKafkaInstanceType"
+    SupportedKafkaInstanceType:
+      description: "Supported Kafka instance type"
+      properties:
+        kind:
+          description: "Indicates the type of this object. Will be 'SupportedKafkaInstanceType' link."
+          type: string
+        id:
+          description: "Unique identifier of the Kafka instance type."
+          type: string
+        sizes:
+          description: " A list of Kafka instance sizes available for this instance type"
+          type: array
+          items:
+            allOf:
+              - $ref: "#/components/schemas/SupportedKafkaSize"
+    SupportedKafkaSize:
+      description: "Supported Kafka Size"
+      type: object
+      properties:
+        kind:
+          description: "Indicates the type of this object. Will be 'SupportedKafkaSize' link."
+          type: string
+        id:
+          description: "Unique identifier of this Kafka instance size."
+          type: string
+        ingressThroughputPerSec:
```

Comment:
I think the naming strategy should be `snake_case` i.e `ingress_throughput_per_sec` instead of `ingressThroughputPerSec`. 
This has to be applied throughout.

Reply:
Thanks @machi1990